airbyte-integrations/connectors/destination-iceberg/src/test-integration/resources/rest-catalog-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
```

Comment:
This is a slimmed down version of Iceberg's quickstart example: https://iceberg.apache.org/spark-quickstart/